bin/vue-build

```diff
@@ -0,0 +1,243 @@
+  performance: {
+    hints: false
+  },
+  resolve: {
+    extensions: ['.js', '.vue', '.css'],
+    modules: [
+      process.cwd(),
+      path.join(process.cwd(), 'node_modules'), // modules in cwd's node_modules
+      path.join(__dirname, '../node_modules') // modules in package's node_modules
+    ],
+    alias: {}
+  },
+  resolveLoader: {
+    modules: [
+      path.join(process.cwd(), 'node_modules'), // loaders in cwd's node_modules
+      path.join(__dirname, '../node_modules') // loaders in package's node_modules
+    ]
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        loader: 'buble-loader',
+        exclude: [/node_modules/]
+      },
+      {
+        test: /\.vue$/,
+        loader: 'vue-loader',
+        options: {
+          postcss: postcssOptions,
+          loaders: Object.assign({
+            js: 'buble-loader'
+          }, loaders.cssLoaders(cssOptions))
+        }
+      }
+    ].concat(loaders.styleLoaders(cssOptions))
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: path.join(__dirname, '../lib/template.html')
+    }),
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
+    }),
+    new webpack.LoaderOptionsPlugin({
+      options: {
+        context: process.cwd(),
+        postcss: postcssOptions,
+        buble: {
+          objectAssign: 'Object.assign',
+          transforms: {
+            dangerousForOf: false,
+            modules: false
+          }
+        }
+      }
+    })
+  ]
+}
+
+if (/\.vue$/.test(options.entry) || options.component) {
+  webpackConfig.entry.client.push(path.join(__dirname, '../lib/default-entry'))
+  webpackConfig.resolve.alias['your-tasteful-component'] = options.entry
+} else {
+  webpackConfig.entry.client.push(options.entry)
+}
+
+// installed by `yarn global add`
+if (isYarn(__dirname)) {
+  // modules in yarn global node_modules
+  // becuase of yarn's flat node_modules structure
+  webpackConfig.resolve.modules.push(path.join(__dirname, '../../'))
+  // loaders in yarn global node_modules
+  webpackConfig.resolveLoader.modules.push(path.join(__dirname, '../../'))
+}
+
+if (production) {
+  webpackConfig.devtool = 'source-map'
+  webpackConfig.plugins.push(
+    new ProgressPlugin(),
+    new webpack.optimize.UglifyJsPlugin({
+      sourceMap: true,
+      compressor: {
+        warnings: false
+      },
+      output: {
+        comments: false
+      }
+    }),
+    new webpack.LoaderOptionsPlugin({
+      minimize: true
+    }),
+    new ExtractTextPlugin('[name].[contenthash:8].css')
+  )
+} else {
+  webpackConfig.devtool = 'eval-source-map'
+  webpackConfig.entry.client.push(
+    require.resolve('webpack-dev-server/client'),
+    require.resolve('webpack/hot/dev-server')
+  )
+  webpackConfig.plugins.push(
+    new webpack.HotModuleReplacementPlugin(),
+    new FriendlyErrorsPlugin(),
+    new PostCompilePlugin(() => {
+      console.log(`> Open http://localhost:${options.port}`)
```

Comment:
after listen, to open(e.g. [opn](https://github.com/sindresorhus/opn)) with default browser is useful. 😉